### PR TITLE
Feature/error page node type

### PR DESCRIPTION
**Pull request by stephanschuler**

### CODE REVIEW

File: Classes/Configuration/NodeBasedConfiguration.php [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+     * known to the ContentRepository context.
+     *
+     * @return Generator<NodeInterface>
+     */
+    protected function getSiteNodes()
+    {
+        $currentContexts = $this->getContentContexts();
+        foreach ($currentContexts as $contextContext) {
+            $rootNode = $contextContext->getRootNode();
+            $rootNodeVariants = [$rootNode, ... $rootNode->getOtherNodeVariants()];
```
Comment (paxuclus) [inline Classes/Configuration/NodeBasedConfiguration.php]: Are you sure this works? The rootNode will get their dimensions removed by a node:repair plugin: https://github.com/neos/neos-development-collection/blob/8.3/Neos.Neos/Classes/Command/NodeCommandControllerPlugin.php#L294

I would expect the `getOtherNodeVariants()` to always return an empty array.

File: Classes/Command/ErrorPageCommandController.php [hunk 2]
```diff
@@ -20,7 +20,10 @@
 use Neos\Utility\Files;
 use Netlogix\ErrorHandler\Configuration\ErrorHandlerConfiguration;
 use Netlogix\ErrorHandler\Service\ControllerContextFactory;
+use Netlogix\ErrorHandler\Service\DestinationResolver;
 use RuntimeException;
+use Symfony\Component\Yaml\Yaml;
```
Comment (paxuclus) [inline Classes/Command/ErrorPageCommandController.php:25]: We should add `symfony/yaml` the composer.json

**Review by paxuclus — CHANGES_REQUESTED** [review]

File: Classes/Configuration/NodeBasedConfiguration.php (hunk 1 above)
Comment (stephanschuler) [reply Classes/Configuration/NodeBasedConfiguration.php]: If site nodes and side nodes lose their dimension information, `FlowQuery::q($siteNode)` should working. So worst case scenario is `$rootNode->getOtherNodeVariants()` returns an empty array and this query isn't doing much.

But after having refactored `$this->getContentContexts()` to use the contentDimensionCombinator, the collection should contain every site node  in every dimension combination anyway, so `getOtherNodeVariants()` is obsolete anyway.

File: Classes/Command/ErrorPageCommandController.php (hunk 2 above)
Comment (stephanschuler) [reply Classes/Command/ErrorPageCommandController.php:25]: Done.

**Review by paxuclus — APPROVED** [review]

**Pull request merged**